src/custom/state/price/updater.ts

```diff
@@ -151,13 +152,26 @@ export default function FeesUpdater(): null {
   const isOnline = useIsOnline()
   const { validTo } = useOrderValidTo()
 
+  // prevents things like "USDC" being used as an address
```

Comment:
Ah yes, `USDC` token is default now. If I recall correctly it'll be replaced by the address eventually then this should just work